.web-docs/metadata.hcl

```diff
@@ -22,13 +22,15 @@ integration {
   ]
   docs {
     process_docs = true
-    # We recommend using the default readme_location of just `./README.md` here
-    # This projects README needs to document the interface of an integration.
-    #
-    # If you need a separate README from what you will display on GitHub vs
-    # what is shown on HashiCorp Developer, this is totally valid, though!
-    readme_location = "./INTEGRATION_README.md"
-    external_url = "https://github.com/hashicorp/integration-template"
+    # Note that the README location is relative to this file. We recommend
+    # keeping the default value, as the adjacent `compile-to-webdocs` script
+    # will automatically copy the README from the `docs` directory of this
+    # repository to the correct location.
+    readme_location = "./README.md"
+    # If you'd prefer not to publish docs on HashiCorp websites, you can
```

Comment:
I would suggest moving this above the `process_docs` attribute just so one can more easily attach that context to the comment